masterlist.yaml

```diff
@@ -22,9 +22,6 @@ globals:
         str: '[Обсуждение LOOT "(Eng)"](http://forums.bethsoft.com/topic/1513925-rel-loot-thread-17/).'
       - lang: es
         str: '[Forum de LOOT](http://forums.bethsoft.com/topic/1513925-rel-loot-thread-17/).'
-  - type: warn
-    condition: ( file("TaleOfTwoWastelands.esm") ) and not checksum("TaleOfTwoWastelands.esm",8104A5CA)
-    content: 'Need check for Tale of Two Wastelands load order.'
```

Comment:
Global warning is leftover from conversion of BOSSv2 masterlist into LOOT masterlist (was introduced by me for BOSSv2)